editoast/src/views/projection.rs

```diff
@@ -518,8 +550,8 @@ pub fn project_train_path_op(
     let mut curve = SpaceTimeCurve::default(); // The current curve being built
     for (a, b) in matching_ops.into_iter().tuple_windows() {
```

Comment:
nitpick: `matching_ops` is already an iterator if I'm not wrong


```suggestion
    for (a, b) in matching_ops.tuple_windows() {
```